bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/i18n/tapocontrol.properties

```diff
@@ -1,89 +1,102 @@
-channel-group-type.tapocontrol.energyMonitor.description = Energy and Power usage
-channel-group-type.tapocontrol.lightBulb.label = Light Bulb
-channel-group-type.tapocontrol.lightBulb.description = Tapo Smart Light Bulb
-channel-group-type.tapocontrol.lightEffect.label = Lightning Effect
-channel-group-type.tapocontrol.lightEffect.description = Tapo Lightning Effects
-channel-group-type.tapocontrol.lightStrip.label = Color Light Strip
-channel-group-type.tapocontrol.lightStrip.description = Tapo Multicolor Smart Light Strip
-channel-group-type.tapocontrol.smartPlug.label = SmartPlug
-channel-group-type.tapocontrol.smartPlug.description = Tapo Smart Plug Power Outlet
-
-# channel types
-
-channel-type.tapocontrol.actualPowerChannel.label = Power
-channel-type.tapocontrol.actualPowerChannel.description = Actual power usage
-channel-type.tapocontrol.colorChannel.label = Color
-channel-type.tapocontrol.colorChannel.description = Color
-channel-type.tapocontrol.colorTemperature.label = Color Temperature
-channel-type.tapocontrol.colorTemperature.description = This channel supports adjusting the color temperature from 2700K to 6500K.
-channel-type.tapocontrol.customEffect.label = Custom Effect
-channel-type.tapocontrol.customEffect.description = Use custom lightning effect
-channel-type.tapocontrol.dimmerChannel.label = Brightness
-channel-type.tapocontrol.dimmerChannel.description = Brightness
-channel-type.tapocontrol.effectName.label = Effect Name
-channel-type.tapocontrol.effectName.description = Name of LightningEffect
-channel-type.tapocontrol.effectOn.label = Lightning Effect Enable
-channel-type.tapocontrol.effectOn.description = Switches the lightning effect on/off
-channel-type.tapocontrol.fade.label = Fade Light
-channel-type.tapocontrol.fade.description = Make the light darker or lighter slowly
-channel-type.tapocontrol.led.label = Switch Led
-channel-type.tapocontrol.led.description = Switch the Smart Home device led on or off.
-channel-type.tapocontrol.lightOn.label = Light On
-channel-type.tapocontrol.lightOn.description = Switches the light on/off
-channel-type.tapocontrol.ontime.label = On-Time
-channel-type.tapocontrol.ontime.description = Number of seconds since the device was powered on
-channel-type.tapocontrol.outputChannel.label = Output Switch
-channel-type.tapocontrol.outputChannel.description = Switches the power state on/off
-channel-type.tapocontrol.overheated.label = Device Overheated
-channel-type.tapocontrol.overheated.description = ON if device is overheated
-channel-type.tapocontrol.todayEnergyUsageChannel.label = Today Usage
-channel-type.tapocontrol.todayEnergyUsageChannel.description = Today energy usage
-channel-type.tapocontrol.todayRuntimeChannel.label = Today Runtime
-channel-type.tapocontrol.todayRuntimeChannel.description = Today runtime (On-Time)
+# binding
+
+binding.tapocontrol.name = TapoControl Binding
+binding.tapocontrol.description = Control your TAPO-SmartHome Devices

```

Comment:
Something went wrong with this file, maybe a wrong file encoding ?

Reply:
Don't really know what went wrong this time.
So i did a clean checkout of main and made my changes there. Now the strange files are correct too.